src/Tribe/Shortcodes/User_Event_Confirmation_List.php

```diff
@@ -65,10 +65,21 @@ public function generate( $params ) {
 	 * @param $params
 	 */
 	protected function set_params( $params ) {
-		$this->params = shortcode_atts( array(
-			'limit' => -1,
+		/**
+		 * Allow filtering of the default limit for the [tribe-user-event-confirmations] shortcode.
+		 * @since TBD
+		 *
+		 * @param int $default_limit The default limit to use.
+		 */
+		$default_limit = apply_filters( 'tribe_tickets_shortcodes_attendee_list_limit', 100 );
+
+		$this->params = shortcode_atts( [
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 4ed7f6bd895a40c96876d6077db5b2d6</sub>